src/page-modules/assistant/trip/index.tsx

```diff
@@ -1,39 +1,33 @@
 import {
-  formatLocaleTime,
   formatToSimpleDate,
   formatToWeekday,
-  isInPast,
   parseIfNeeded,
 } from '@atb/utils/date';
 import {
   TripData,
-  type TripPattern,
-} from '@atb/page-modules/assistant/server/journey-planner/validators';
+  type TripPattern as TripPatternType,
+} from '../server/journey-planner/validators';
 import style from './trip.module.css';
 import { PageText, useTranslation } from '@atb/translations';
 import { Typo } from '@atb/components/typography';
-import { motion } from 'framer-motion';
-import { MonoIcon } from '@atb/components/icon';
-import { TransportIconWithLabel } from '@atb/components/transport-mode/transport-icon';
 import { GeocoderFeature } from '@atb/page-modules/departures';
 import { TransportModeFilterOption } from '@atb/components/transport-mode-filter/types';
-import { nextTripPatterns } from '@atb/page-modules/assistant/client';
+import { nextTripPatterns } from '../client';
 import {
   DepartureMode,
   NonTransitTripData,
   createTripQuery,
   filterOutDuplicates,
   getCursorByDepartureMode,
-} from '@atb/page-modules/assistant';
+} from '..';
```

Comment:
Is this now essentially circular dependency? 🤔

Reply:
Good question 🤔

Reply:
Thought about this when I moved it all. Maybe the utils.ts should move to the trip folder now?

Reply:
Looked into moving the utils to `trip/`, but several of the util functions are used in several places, e.g. `layout.tsx` and `assistant/trip/index.tsx`, or `assistant/trip/index.tsx` and `journey-planner/index.tsx`. How should this be solved?

Reply:
Thnigs inside the assistant directory shouldn't require things from index, but have index expose the API externally. If its utils I'd place it in a utils file along side index.

Reply:
I think it is in a utils folder. Removing the export from index. 

It should be rare that a page-module exports something that's used elsewhere, other than in `pages` right?

Reply:
Yeah, I think exported things for page-modules should only be the corresponding page. If imported by something else it is not page-module